### DISCUSSION TITLE
I wish Tags exported with .md /html

### DISCUSSION THREAD

**Discussion by drhus** [standardnotes/forum#3362] [Suggestions]
When exporting a note as .md you get the note as markdown (including links to another notes which is great) but no tags, as tags becoming part of SN native i expect tags to be exported

**Comment by effieeee** [comment]
Hi @drhus, tags are likely part of the items that are lost while exporting, especially since it's an inbuilt feature that's specific to the app.
![image](https://github.com/standardnotes/forum/assets/54989748/ae173553-c808-47a5-8388-d7489fd440f5)
